apps/frontend/core-dapp/src/components/Intercom/IntercomButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, ReactNode, Suspense, useEffect } from 'react'
+import { useIntercom } from './index'
+
+type Props = {
+  children: (params: { onClick: () => void }) => ReactNode | undefined
+}
+
+const IntercomButton: FC<Props> = ({ children }) => {
+  const intercom = useIntercom()
+
+  useEffect(() => {
+    if (!intercom) return () => {}
+
+    intercom.setWidgetVisibility(false)
+
+    return () => {
+      intercom.setWidgetVisibility(true)
+    }
+  }, [intercom])
+
+  if (!intercom) return null
+
+  return <>{children({ onClick: intercom.show })}</>
+}
+
+const Wrapper: FC<Props> = ({ children }) => (
+  <Suspense fallback={<>{/* nothing */}</>}>
```

Comment:
I believe you could use null to achieve the same thing:
```suggestion
  <Suspense fallback={null}>
```

Reply:
Apparently can also remove it entirely haha